antiplag/migrations/0014_migrate_uuid_foreign_keys.py

```diff
@@ -0,0 +1,50 @@
+from django.db import migrations, models
+
+
+def populate_submission_fk(apps, schema_editor):
+    Document = apps.get_model("antiplag", "Document")
+    for document in Document.objects.all():
+        document.submission_id = document.submission_uuid
+        document.save()
+
+
+def populate_document_fk(apps, schema_editor):
+    Result = apps.get_model("antiplag", "Result")
+    for result in Result.objects.all():
+        result.document_id = result.document_uuid
+        result.save()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("antiplag", "0013_make_uuid_fields_pk"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="Document",
+            name="submission",
+            field=models.ForeignKey(
+                on_delete=models.CASCADE,
+                null=True,
+                related_name="documents",
+                to="antiplag.Submission",
+            ),
+        ),
+        migrations.AddField(
+            model_name="Result",
+            name="document",
+            field=models.OneToOneField(
+                on_delete=models.CASCADE,
+                null=True,
```

Comment:
Looks like you forgot final migration to remove `null=True`.

Reply:
You're right, added one last migration.